CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Contributing guidelines](#contributing-guidelines)
+    - [Sign the CLA](#sign-the-cla)
+    - [Contributing A Patch](#contributing-a-patch)
+    - [Issue and Pull Request Management](#issue-and-pull-request-management)
+    - [Pre-check before submitting a PR](#pre-check-before-submitting-a-pr)
+    - [Build and push images](#build-and-push-images)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Contributing guidelines
+
+## Sign the CLA
+
+IBM cloud projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests.  Please see xxx for more info
```

Comment:
we will avoid mention CLA, RH uses DCO more. here is an example https://github.com/open-cluster-management/api/pull/65

Reply:
good point... thx.